### PR TITLE
Clean up unused code

### PR DESCRIPTION
**Pull request by mdoucet**
There are reduction options that are in Refred and put into the out xml file that are never used. Those should be cleaned up.
Strange constant file should also be removed. This is an incremental attempt to clean code.

### CODE REVIEW

File: RefRed/calculations/compare_two_lrdata.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import RefRed.constants
+THETA_TOLERANCE = 0.05
```
Comment (jmborr) [inline RefRed/calculations/compare_two_lrdata.py:1]: Should this be stored under one of the modules in `config/`?

File: RefRed/calculations/lr_data.py [hunk 2]
```diff
@@ -16,6 +15,7 @@
 
 # any run before, tmin and tmax will be used a different algorithm
 RUN_NUMBER_0_BETTER_CHOPPER_COVERAGE = 137261
+NEW_GEOMETRY_DATE = '2014-10-01'
```
Comment (jmborr) [inline RefRed/calculations/lr_data.py:18]: same question as before

File: RefRed/plot/single_click_plot.py [hunk 3]
```diff
@@ -1,13 +1,14 @@
...
-
-import RefRed.constants
 from RefRed.plot.popup_plot_1d import PopupPlot1d
 from RefRed.plot.popup_plot_2d import PopupPlot2d
 from RefRed.plot.launch_stitching_manual_axis import LaunchStitchingManualXAxis
 from RefRed.plot.launch_stitching_manual_axis import LaunchStitchingManualYAxis
 from RefRed.gui_handling.gui_utility import GuiUtility
 
 
+DOUBLE_CLICK_DELAY = 0.4  # seconds
```
Comment (jmborr) [inline RefRed/plot/single_click_plot.py:9]: same, looks like a lot of hard-coded constants which may be better managed if in one place rather that spread-out over the code.

**Review by jmborr — CHANGES_REQUESTED** [review]

File: RefRed/calculations/compare_two_lrdata.py (hunk 1 above)
Comment (mdoucet) [reply RefRed/calculations/compare_two_lrdata.py:1]: I think that putting parameters in a python file isn't very clean. I think that if parameters are coded in a python file, they should be with the code that uses it. If they are real parameters that can be changes, then they should be in a loadable json file (or similar editable settings file). Let's chat about it.

Comment (mdoucet) [reply RefRed/calculations/compare_two_lrdata.py:1]: OK I think I've been convinced by someone else about a better approach :)

Comment (jmborr) [reply RefRed/calculations/compare_two_lrdata.py:1]: a config file?

**Pull request merged**